esy/bin/esyCommand.ml

```diff
@@ -341,7 +341,16 @@ let build ?(buildOnly=true) cfg command =
         cfg task
     in Build.buildTask ~force:true ~stderrout:`Keep ~quiet:true ~buildOnly cfg task
 
-  | command ->
+  | argv ->
```

Comment:
Let's do `cmd::args` here so we don't have to use unsafe `List.tl` and we can use `cmd` to lookup the script.